numpy/core/src/umath/ufunc_object.c

```diff
@@ -5893,13 +5893,53 @@ new_array_op(PyArrayObject *op_array, char *data)
     return (PyArrayObject *)r;
 }
 
+/*
+ * Use an indexed loop to do the work
+ * Returns 0 if successful
+ */
+static int
+trivial_at_loop(PyArrayMethodObject *ufuncimpl, NPY_ARRAYMETHOD_FLAGS flags,
+                    PyArrayMapIterObject *iter,
+                    PyArrayObject *op1_array, PyArrayObject *op2_array,
+                    PyArrayMethod_Context *context)
+{
+    int buffersize=0, errormask = 0;
+    int res;
+    char *args[3];
+    const char * ufunc_name = ufunc_get_name_cstr((PyUFuncObject *)context->caller);
```

Comment:
Is this needed up front? I guess normally `buffersize` would be needed, but not here. So, might it be faster to only do this inside the `res == 0` stanza (I guess that only helps the case that one is ignoring float point errors, but maybe nice if that is slightly faster...)

Reply:
ok